pkg/mcs/resource_manager/server/manager.go

```diff
@@ -66,10 +64,10 @@ func NewManager(srv bs.Server) *Manager {
 			kv.NewEtcdKVBase(srv.GetClient(), "resource_group"),
 			nil,
 		)
-		m.member = srv.GetMember()
+		m.srv = srv
 	})
-	// The second initialization after the leader is elected.
-	srv.AddLeaderCallback(m.Init)
+	// The second initialization after becoming serving.
+	srv.AddServiceReadyCallback(m.Init)
```

Comment:
Can we merge `AddStartCallback` and `AddServiceReadyCallback`?

Reply:
It will affect pd legacy server.